### DISCUSSION TITLE
Unable to link to specific line number when using the New Issue Form Template

### DISCUSSION THREAD

**Discussion by hyperupcall** [community/community#40598] [Projects and Issues]
### Select Topic Area

Bug

### Body

I love the new abilities of the issue Form Template. For me, it makes the flow of contributing to new codebases much easier. In doing so though, I think I found a rough edge: the lack of an ability to highlight a specific line of a new issue template.

For example, in [this](https://github.com/typescript-eslint/typescript-eslint/blob/bed6356866c04e6e4d5db3f71f31583dbbe0f36c/.github/ISSUE_TEMPLATE/06-bug-report-other.yaml) issue template, I wanted to reference a specific line that should be fixed (to fix a 404 URL).

Normally, the file will have line numbers, and clicking on one will show three dots to grab the link to the specific line number of the file

But, the new issue template shows how it will display to new users (which I think is useful!) - and there is no obvious option to get a hyperlink to a specific line. I clicked the "three dots" at in the corner and clicked "Go to line", but that didn't seem to do anything.

A potential fix for my case would have like a "Code" button (right in between the _Raw_ and _Blame_ buttons), that shows the traditional view of the file

## Miscellaneous Issue

Also a sort of miscellaneous thing, I navigated to this repository from the blue popup with the green "Beta" chip that said:

> (Beta) This file is used as an Issue Form template. [Give Feedback.](https://github.com/github/feedback/discussions/categories/issues-feedback)

But, it seems the correct url ends with [/issues](https://github.com/community/community/discussions/categories/issues)?. This experience made me feel confused and I felt discouraged from creating an new discussion since it wasn't clear to me what to do next

**Comment by ossanna16** [comment]
Hi there @hyperupcall and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
